include/class.plugin.php

```diff
@@ -539,7 +539,7 @@ static function isVerified($phar) {
         elseif (!file_exists(INCLUDE_DIR . '/plugins/updates.pem'))
             return self::VERIFY_NO_KEY;
 
-        if (!isset($pubkey)) {
+        if (!isset($pubkey) && extension_loaded('openssl')) {
```

Comment:
I know this technically resolves the issue but if you look closely we end up returning "wrong" exit code when a more suitable one, `self::VERIFY_EXT_MISSING`, is available. This is especially important if UI is (or will be) dependent on the returned value. I would refactor and put the check around line 537 as shown below. 

```php
       if (!class_exists('Phar') || !extension_loaded('openssl'))
            return self::VERIFY_EXT_MISSING;
```